package.json

```diff
@@ -31,10 +29,9 @@
     "release": "standard-version",
     "server": "serve docs --listen 3000",
     "start": "npm-run-all --parallel build:css watch:*",
-    "test": "npm-run-all css",
```

Comment:
... so it's the same as `css`, `css-compile`, and `build:css` - and _not_ what the README claimed it to be.

Removed. It wasn't testing anyway.